### ISSUE TITLE
Fix Hardcoded icon in scudcloud.desktop

### ISSUE THREAD

**Issue by happydemic** [raelgc/scudcloud#456]
scudcloud 1.25 (in Ubuntu 16.04)

Hi - thanks for the great program. I'm still enjoying it, though I have tried out the official client too. ;)

One quick request - the ScudCloud application icon is [hardcoded](https://github.com/Foggalong/hardcode-fixer) - see [listing](https://github.com/Foggalong/hardcode-fixer/wiki/App-Support). 

This is a one-line fix in the .desktop file to
`Icon=scudcloud`
(from current `Icon=/usr/share/pixmaps/scudcloud.png`)

This allows the user's icon theme to display its own scudcloud icon when that theme includes one (e.g. Numix Circle). 

Note: Numix Circle includes both scudcloud.svg and ScudCloud.svg. If the app icon were set to 'ScudCloud' rather than 'scudcloud' it would be possible for icon theme designers to use different icons for the indicator (scudcloud) and the application (e.g. ScudCloud), but that would require you to add a new `ScudCloud.png` icon to your default icon sets.

Thanks!

**Comment by stieg** [comment]
Hi @happydemic ,

   Thanks for the write up on why folks do that.  I'll play around with this soon and see what it would take.  Cheers.

**Comment by raelgc** [comment]
Just sharing a concern about this:

We're using the themed icon for the systray icon. When we try remove the hardcoded icon from the `.desktop` file, it'll use the themed icon, which will be the monocolor icon in Unity.

So, this is the reason why we used the hardcoded icon: we don't want to use the monocolor icon in the `.desktop` file.

One idea is to check if we can create another resource name to refer in the `.desktop` file.

Two workarounds: create your own `.desktop` file under `~/.local/.local/share/applications/` or use `dpkg-divert` to change the system `.desktop` file and preserve it even under system update.

**Comment by happydemic** [comment]
@raelgc Yep, exactly. In my own .local .desktop file, I use `Icon=ScudCloud` for exactly that reason. I have then added a colour ScudCloud.png or ScudCloud.svg to those themes which inherit the ubuntu-mono indicator icons.
